consumers/hidden-test-consumer/src/hidden_test_consumer/hidden_test_process_pb2_grpc.py

```
@@ -0,0 +1,97 @@
+                    request_deserializer=hidden__test__process__pb2.ProcessRequest.FromString,
+                    response_serializer=hidden__test__process__pb2.ProcessResponse.SerializeToString,
+            ),
+    }
+    generic_handler = grpc.method_handlers_generic_handler(
+            'hidden_test_process.HiddenTestProcess', rpc_method_handlers)
+    server.add_generic_rpc_handlers((generic_handler,))
+    server.add_registered_method_handlers('hidden_test_process.HiddenTestProcess', rpc_method_handlers)
+
+
+ # This class is part of an EXPERIMENTAL API.
+class HiddenTestProcess(object):
```

Ruff (0.8.2)

Class `HiddenTestProcess` inherits from `object`

Remove `object` inheritance

(UP004)